### PR TITLE
124 make top navbar

### PR DESCRIPTION
**Pull request by croueAlec**

### CODE REVIEW

**Review by Neffi42 — APPROVED** [review]

**Review by Neffi42 — CHANGES_REQUESTED** [review]
No no no...

File: django/src/pages/templates/extends/base.html [hunk 1]
```diff
@@ -32,4 +58,8 @@
 			route(event.target.getAttribute('href'));
 		});
 	});
+
+	document.getElementById("logoutButtonNavbar").addEventListener("click", function (e) {
```
Comment (antoineverin) [inline django/src/pages/templates/extends/base.html:62]: rename id to respect naming norm `logout-button-navbar`

File: nginx/src/css/navbar.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.navbar {
+	background-color: #424242;
+	color: #fecaca;
+	height: 4.2rem;
+	min-height: 60px;
```
Comment (antoineverin) [inline nginx/src/css/navbar.css:5]: avoid using `px`

**Review by antoineverin — CHANGES_REQUESTED** [review]

**Review by antoineverin — APPROVED** [review]

File: nginx/src/css/navbar.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+.navbar-brand {
+	color: white;
+	padding-left: 5rem;
+}
+
+.div-navbar-profile {
+	display: inline;
+	padding-right: 4.5;
```
Comment (antoineverin) [inline nginx/src/css/navbar.css:16]: missing unit

**Review by antoineverin — CHANGES_REQUESTED** [review]

**Review by antoineverin — APPROVED** [review]

**Review by Neffi42 — APPROVED** [review]

**Pull request merged**